sessionctx/variable/sysvar.go

```diff
@@ -1325,6 +1325,15 @@ var defaultSysVars = []*SysVar{
 		s.EnablePaging = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBPersistAnalyzeOptions, Value: BoolToOnOff(DefTiDBPersistAnalyzeOptions), skipInit: true, Type: TypeBool,
+		GetGlobal: func(s *SessionVars) (string, error) {
+			return strconv.FormatBool(PersistAnalyzeOptions.Load()), nil
```

Comment:
Maybe we can use `BoolToOnOff` instead of `strconv.FormatBool`

Reply:
Done